Mathlib/Algebra/Order/Group/Defs.lean

```diff
@@ -86,14 +82,14 @@ theorem exists_one_lt' [Nontrivial α] : ∃ a : α, 1 < a := by
   · exact ⟨y, h⟩
 
 -- see Note [lower instance priority]
-@[to_additive]
+@[to_additive LinearOrderedAddCommGroup.to_noMaxOrder]
```

Comment:
I'm surprised `to_additive` suggested the name of the original lemma again here and below. I guess it doesn't know that `CommGroup` should become `AddCommGroup` if it starts with `LinearOrdered`? Possibly something to look into after #29352.

cc: @fpvandoorn, @JovanGerb

Reply:
The correct incantation is 
```lean
run_meta ToAdditive.insertTranslation `LinearOrderedCommGroup `LinearOrderedAddCommGroup
```
I'll update the `ToAdditive` doc.